README.md

```diff
@@ -24,6 +24,6 @@ To build the Calico Networking Plugin for CNI locally, clone this repository and
 
 [cni]: https://github.com/appc/cni
 [config]: configuration.md
-[calico-containers]: https://github.com/projectcalico/calico-containers/blob/master/docs/cni/kubernetes/README.md
+[calico-containers]: https://github.com/projectcalico/calico-containers/blob/master/README.md
```

Comment:
@caseydavenport would it be http://docs.projectcalico.org/master/reference/advanced/calico-cni or http://docs.projectcalico.org/master/getting-started/kubernetes/ ?